src/Propel/Runtime/ActiveQuery/SqlBuilder/AbstractSqlQueryBuilder.php

```diff
@@ -127,7 +127,7 @@ public function buildParams(array $columnNames, ?Criteria $values = null): array
      * Build sql statement from a criteria and add it to the given statement collector.
      *
      * @param \Propel\Runtime\ActiveQuery\Criterion\AbstractCriterion $criterion
-     * @param array<mixed>|null $params
+     * @param array|null $params
```

Comment:
Thats a regression IMO
We should keep the more specific annotation where we have them to signal the other places where we didnt check yet
The stan tools are wrong here to assume that those are equal, they are not in semantic meaning.